aom/src/main/java/com/nedap/archie/xml/adapters/TimeIntervalXmlAdapter.java

```diff
@@ -1,9 +1,11 @@
 package com.nedap.archie.xml.adapters;
 
+import java.time.temporal.TemporalAccessor;
+
 /**
  * Created by pieter.bos on 28/07/16.
  */
-public class TimeIntervalXmlAdapter extends AbstractIntervalAdapter {
+public class TimeIntervalXmlAdapter extends AbstractIntervalAdapter<String, TemporalAccessor> {
```

Comment:
It won't add much information, but it will make the Java compile work differently. When generics are missing or incomplete, it will disable generics validation for the whole statement and issue a warning. For public methods and types, this warnings will also show up in programs that use archie.

I also fixed most of the Raw use of parameterized class warnings in archie itself to make sure our generics were defined correctly.